.gitlab-ci.yml

```diff
@@ -33,6 +33,9 @@ test:
 
 publish latest:
   <<: *publish
+  only:
+    - tags
+    - master
```

Comment:
For this to work I think we have to disable merge commits otherwise what will go to `master` will always be one ahead of the tag :]

Reply:
Is that a problem though?